Source/WebCore/testing/Internals.cpp

```diff
@@ -1670,7 +1674,7 @@ ExceptionOr<Vector<AtomString>> Internals::formControlStateOfPreviousHistoryItem
 
 ExceptionOr<void> Internals::setFormControlStateOfPreviousHistoryItem(const Vector<AtomString>& state)
 {
-    HistoryItem* mainItem = frame()->loader().history().previousItem();
+    auto* mainItem = frame() ? frame()->loader().history().previousItem() : nullptr;
```

Comment:
I’d think this would need to be `RefPtr` instead of `auto*` in our new safer C++ model.

Reply:
Done